app/models/user.rb

```diff
@@ -29,15 +29,15 @@ class User < ActiveRecord::Base
          :rememberable, :trackable, :validatable
   devise :omniauthable, :omniauth_providers => [:google_oauth2]
 
-  has_many :appreciations_received, foreign_key: "recipient_id", class_name: "Appreciation"
-  has_many :appreciations_sent, foreign_key: "submitter_id", class_name: "Appreciation"
-  has_many :messages_sent, foreign_key: "submitter_id", class_name: "Message"
-  has_many :messages_received, foreign_key: "recipient_id", class_name: "Message"
+  has_many :appreciations_received, foreign_key: "recipient_id", class_name: Appreciation
+  has_many :appreciations_sent, foreign_key: "submitter_id", class_name: Appreciation
+  has_many :messages_sent, foreign_key: "submitter_id", class_name: Message
```

Comment:
[pronto-rubocop warning] Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.